samza-core/src/main/scala/org/apache/samza/checkpoint/OffsetManager.scala

```diff
@@ -535,9 +536,47 @@ class OffsetManager(
         startpoints
           .foreach(taskMap => taskMap._2
             .foreach(sspMap => info("Loaded startpoint: %s for SSP: %s and task: %s" format (sspMap._2, sspMap._1, taskMap._1))))
+        resolveStartpointsToStartingOffsets
```

Comment:
Minor: Would be better to update the java doc of the `OffsetManager` class and include that it accounts for startpoints into calculation of the starting offset.

Reply:
Done. Thanks for the feedback.